cljfmt/src/cljfmt/tool.clj

```diff
@@ -99,25 +100,77 @@
   ([a]   a)
   ([a b] (merge-with + a b)))
 
-(defn check-no-config
-  "The same as `check`, but ignores dotfile configuration."
-  [options]
-  (let [map*   (if (:parallel? options) pmap map)
-        counts (->> (:paths options)
-                    (mapcat (partial find-files options))
-                    (map* (partial check-one options))
-                    (map (fn [status]
-                           (print-file-status options status)
-                           (:counts status)))
-                    (reduce merge-counts))]
+(defn- merge-statuses
```

Comment:
I don't understand the purpose of `merge-statuses`. Can you explain?

Reply:
It's the counterpart to `merge-counts` when you want a fuller report of what was done, which you often do when using this as a library.